docs/_layouts/default.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Code highlighting -->
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/highlight.css">
+
+    <!-- Custom -->
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/sockeye.css">
+
+    <title>{{ site.title }}</title>
+  </head>
+  <body>
+    <div class="container-fluid mt-1">
+      <div class="row">
+        <div class="col-12 col-md-2">
+          <h1 class="text-center title">SOCKEYE</h1>
+
+          <p class="text-center">A toolkit for neural sequence-to-sequence transduction</p>
+
+          <ul>
+            <li><a href="{{ site.baseurl }}/index.html">Home</a>
+            <li><a href="{{ site.baseurl }}/setup.html">Installation</a>
+            <li><a href="{{ site.baseurl }}/training.html">Training</a>
+            <li><a href="{{ site.baseurl }}/inference.html">Inference</a>
+            <li><a href="{{ site.baseurl }}/scoring.html">Scoring</a>
+            <li><a href="{{ site.baseurl }}/development.html">Development</a>
+            <li><a href="{{ site.baseurl }}/tutorials.html">Tutorials</a>
+            <li><a href="{{ site.baseurl }}/image_captioning.html">Image Captioning</a>
+            <li><a href="{{ site.baseurl }}/faq.html">FAQ</a>
```

Comment:
Would it make sense to remove the FAQ entirely? Initially I thought we'd fill it with questions/answers over time. As it is currently, it is not very useful to users. I think the error message in the FAQ is even out-dated and would not be thrown anymore.

Reply:
sure